### PR TITLE
added models for 7.0-3-I6-2

### PR DESCRIPTION
**Pull request by kyoungyun**

### CODE REVIEW

File: vendor/cisco/nx/7.0-3-I6-2/cisco-nx-openconfig-bgp-multiprotocol-deviations.yang [hunk 1]
```diff
@@ -0,0 +1,31 @@
+
```
[the remark this answers is not in the document]
Comment (kyoungyun) [reply vendor/cisco/nx/7.0-3-I6-2/cisco-nx-openconfig-bgp-multiprotocol-deviations.yang]: It's not empty.  There is no deviation so just there's no deviation statement.

[the remark this answers is not in the document]
Comment (einarnn) [reply vendor/cisco/nx/7.0-3-I6-2/cisco-nx-openconfig-bgp-multiprotocol-deviations.yang]: If here are no deviations, there shouldn't be a file. It is misleading IMO.

[the remark this answers is not in the document]
Comment (kyoungyun) [reply vendor/cisco/nx/7.0-3-I6-2/cisco-nx-openconfig-bgp-multiprotocol-deviations.yang]: Ok.  I removed the file with no deviations.

**Comment by kyoungyun** [comment]
The empty deviation file has been removed.

**Pull request merged**